pymongo/pool.py

```diff
@@ -662,7 +662,7 @@ def __init__(self, sock, pool, address, id):
             # This is a Monitor connection.
             self.cancel_context = _CancellationContext()
         self.opts = pool.opts
-        self.more_to_come = False
+        self.more_to_come: Union[int, bool] = False
```

Comment:
why int? IIUC this should be bool

Reply:
Let's make _OpMsg.more_to_come a bool as well.

Reply:
Sounds good